### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by bdwyer2**
These tests don't check for accuracy they just check to make sure the wav file output looks okay.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,17 +6,13 @@ ARG model_files=models.tar.gz
 
 # Get model archive and unzip it to assets folder
 RUN wget -nv ${model_bucket}/${model_files} --output-document=/workspace/assets/${model_files} --show-progress --progress=bar:force:noscroll
-RUN tar -x -C assets/ -f assets/${model_files} -v
+RUN tar -x -C assets/ -f assets/${model_files} -v && rm assets/${model_files}
 
 # Install dependencies for wave file encoding
 RUN apt-get update && apt-get install -y ffmpeg && rm -rf /var/lib/apt/lists/*
```
Comment (ptitzler) [inline Dockerfile]: I don't think we need `ffmpeg` anymore since mp3 support is not included?

File: tests/test.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import pytest
+import requests
+import scipy.io.wavfile
+import numpy
+
+
```
Comment (ptitzler) [inline tests/test.py]: We should add tests (however trivial they might be) for each endpoint.

File: Dockerfile (hunk 1 above)
Comment (bdwyer2) [reply Dockerfile]: Thanks for catching that

File: tests/test.py (hunk 2 above)
Comment (bdwyer2) [reply tests/test.py]: I think we are doing that already? I have a for loop on line 13 that checks every endpoint

Comment (ptitzler) [reply tests/test.py]: Right, but I was thinking about the swagger endpoint `/` and the metadata endpoint `/model/metadata`.

Comment (bdwyer2) [reply tests/test.py]: Ohhh, I understand. Currently none of the other integration tests are testing the `/model/metadata` endpoint, do you think we need to add this to all of the models?

Comment (ptitzler) [reply tests/test.py]: yes please. as trivial as the endpoint implementations might be, it'd be good to get as close to 100% coverage as possible

Comment (ptitzler) [reply tests/test.py]: For the swagger, maybe just check the HTTP code and that the response is JSON? I don't really know how frequently the JSON response changes and how painful it'd be to keep the tests up to date. Your call.

**Pull request merged**